include/queues.h

```diff
@@ -44,7 +44,7 @@ typedef struct st_queue {
   uint offset_to_queue_pos;    /* If we want to store position in element */
   uint auto_extent;
   int max_at_top;	/* Normally 1, set to -1 if queue_top gives max */
-  int  (*compare)(void *, uchar *,uchar *);
+  int  (*compare)(const void *, const void *, const void *);
```

Comment:
put argument names here, please. with all three arguments being of
the same type it's totally unclear what means what. Better:
```c
int  (*compare)(const void *param, const void *a, const void *b);
```